cint/cint.py

```diff
@@ -58,13 +66,18 @@ def from_address(cls, address):
     # Cint details below
     @classmethod
     def __stronger_type(cls, other):
-        if isinstance(other, (int, long)):
+        if isinstance(other, (int, long, float)):
             return cls
-        elif not isinstance(other, INTS):
+        elif not isinstance(other, (INTS, FLOATS)):
             raise ValueError("Cannot perform arithmetic operations between %s and %s" % (cls, type(other)))
 
         other = other.__class__
 
+        # FLOATS are *stronger* than INTS
+        if (cls in INTS and other in FLOATS) or (cls in FLOATS and other in INTS):
+            return other if cls in INTS else cls
```

Comment:
the upper scope if is not so readable, maybe it should be `if ... elif`, I'll think about it more later.